samples/wifi/shell/README.rst

```diff
@@ -430,7 +430,7 @@ To test the SAP mode, the sample must be built using the configuration overlay :
 
    .. code-block:: console
 
-      wifi reg_domain IN
+      wifi reg_domain US
```

Comment:
Why is this change needed? I intentionally kept it as IN, 5GHz should work with IN.

Reply:
Commit log need to be reworded? What is `display`?

Reply:
> Why is this change needed? I intentionally kept it as IN, 5GHz should work with IN.

There seems to be a mismatch in the documentation where it mentions setting the regulatory domain to `US` but provides the command for `IN`, creating confusion.

Reply:
As you kept IN intentionally, will edit the line which is mentioning about how to set regualtory in the document.

Reply:
addressed.